src/Orangebeard.Client/OrangebeardV2Client.cs

```diff
@@ -0,0 +1,257 @@
+        }
+
+        public OrangebeardV2Client(string endpoint, Guid uuid, string projectName, string testSetName, bool connectionWithOrangebeardIsValid) : this()
+        {
+            config = new OrangebeardConfiguration(endpoint, uuid, projectName, testSetName);
+            Init(endpoint, uuid);
+            this.connectionWithOrangebeardIsValid = connectionWithOrangebeardIsValid;
+        }
+
+        public OrangebeardV2Client(OrangebeardConfiguration config, bool connectionWithOrangebeardIsValid) : this()
+        {
+            this.config = config;
+            Init(config.Endpoint, new Guid(config.AccessToken));
+            this.connectionWithOrangebeardIsValid = connectionWithOrangebeardIsValid;
+        }
+
+        #region Start, update, and finish test runs.
+
+        public override Guid? StartTestRun(StartTestRun testRun)
+        {
+            if (connectionWithOrangebeardIsValid)
+            {
+                try
+                {
+                    httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", config.AccessToken.ToString());
+                    Uri uri = new Uri($"{config.Endpoint}/listener/v2/{config.ProjectName}/launch");
+                    string json = JsonConvert.SerializeObject(testRun);
+                    StringContent content = new StringContent(json, System.Text.Encoding.UTF8, AbstractClient.APPLICATION_JSON);
+
+                    var result = httpClient.PostAsync(uri, content).Result;
+
+                    string jsonResponseString = result.Content.ReadAsStringAsync().Result;
+                    JObject jsonResponseObject = JObject.Parse(jsonResponseString);
+                    string id = jsonResponseObject.Value<string>("id");
+                    return new Guid(id);
+                }
+                catch (Exception)
+                {
+                    //LOGGER.LogError("The connection with Orangebeard could not be established! Check the properties and try again!");
+                    connectionWithOrangebeardIsValid = false;
+                }
+            }
+            return null;
+        }
+
+        public override void UpdateTestRun(Guid testRunUUID, UpdateTestRun updateTestRun)
+        {
+            if (connectionWithOrangebeardIsValid)
+            {
+                httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", config.AccessToken.ToString());
+                Uri uri = new Uri($"{config.Endpoint}/listener/v2/{config.ProjectName}/launch/{testRunUUID}/update");
+                String json = JsonConvert.SerializeObject(updateTestRun);
+                StringContent content = new StringContent(json, System.Text.Encoding.UTF8, AbstractClient.APPLICATION_JSON);
+                _ = httpClient.PutAsync(uri, content).Result;
+            }
+            else
+            {
+                //LOGGER.LogWarning("The connection with Orangebeard could not be established!");
```

Comment:
remove or actually log in case of misconfig?

Reply:
Log it using log4j.